PetrolEngine.cpp

```diff
@@ -1,20 +1,21 @@
-#include "PetrolEngine.hpp"
 #include <iostream>
 
+#include "PetrolEngine.hpp"
+#include "exceptions.hpp"
+
 PetrolEngine::PetrolEngine(int power, float capacity, int gears)
-    : power_(power)
-    , capacity_(capacity)
-    , gears_(gears)
-    , currentGear_(0)
-{
+    : power_(power), capacity_(capacity), gears_(gears), currentGear_(0) {
     std::cout << __FUNCTION__ << std::endl;
 }
 
-void PetrolEngine::changeGear(int gear)
-{
-    // TODO: Add checking if gear is between -1 and gears_
-    // -1 is for REAR
-    // 0 is for NEUTRAL
+void PetrolEngine::changeGear(int gear) {
+    if (gear < -1 || gear > gears_) {
```

Comment:
I suggest you to avoid magic numbers like -1. Maybe make -1 as constexpr

Reply:
OFC Thanks :)